cmd/detectExecuteScan.go

```diff
@@ -145,12 +145,12 @@ func addDetectArgs(args []string, config detectExecuteScanOptions, utils detectU
 	args = append(args, fmt.Sprintf("--blackduck.api.token=%v", config.Token))
```

Comment:
No `\"` here?

Reply:
ApiToken created from BlackDuck is not supposed to have spaces (or other special characters). Hence we do not need to escape these characters here. If the user provides a value with illegal characters then the command should fail before triggering the scan.